src/store/socket_init.js

```diff
@@ -1,31 +1,77 @@
+/* eslint no-param-reassign: ["error", { "props": false }] */
 import io from 'socket.io-client';
-import * as actions from './Chat/actions.js';
-// import actions that have anything to do with sockets.
+import {
```

Comment:
yeah that makes sense. If it bothered us enough, we could group the socket actions into a object and export that as well.